### DISCUSSION TITLE
A way to name the .wasm

### DISCUSSION THREAD

**Discussion by BlackAsLight** [wasm-bindgen/wasm-bindgen#3465] [Q&A] [answered]
### Summary

Is there a way to name the .wasm file? I noticed it takes the name of the input .wasm file and appends _bg to the name. I'd wondering if I could explicitly dictate what name it should give exactly. Simply changing the file name wont work as the corresponding .js file created relies on the _bg name.

**Comment by Liamolucko** [comment, marked as answer]
You can't specify the _exact_ name, but you can use `--out-name` to choose the bit before the `_bg`. For example, if you specified `--out-name foo`, the output files would be named `foo.js` and `foo_bg.wasm`.
